src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,454 @@
+"""
+This script takes in appropriate datasets as input and runs the MEDUSA2.0 model to calculate
+TSS (total suspended solids), TCu (total copper), DCu  (dissolved copper), TZn (total zinc), and DZn (dissolved zinc).
+"""
+
+import logging
+import math
+from enum import StrEnum
+
+import geopandas as gpd
+import numpy as np
+import xarray as xr
+from sqlalchemy.engine import Engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.sql import text
+
+from src.digitaltwin import setup_environment
+from src.digitaltwin.utils import LogLevel, setup_logging, get_catchment_area
+
+log = logging.getLogger(__name__)
+
+Base = declarative_base()
+
+
+# Enum strings are assigned as they are described in the original paper
+class SurfaceType(StrEnum):
+    CONCRETE_ROOF = "Cr"
+    COPPER_ROOF = "Cu"
+    GALVANISED_ROOF = "Gv"
+    ASPHALT_ROAD = "Rd"
+    CAR_PARK = "CrP"  # CarParks are classified the same as roads
+
+
+def compute_tss_roof_road(surface_area,
+                          antecedent_dry_days,
+                          average_rain_intensity,
+                          event_duration,
+                          surface_type):
+    """
+    Calculates the total suspended solids (TSS) given the following parameters;
+
+    Parameters
+    ----------
+    surface_area: float
+        surface area of the given surface type
+    antecedent_dry_days: float
+        length of antecedent dry period (days)
+    average_rain_intensity: float
+        average rainfall intensity of the event (mm/h)
+    event_duration: float
+        duration of the rainfall event (h)
+    surface_type: int
+        the type of surface we are computing the TSS for
+
+    Returns
+    -------
+    float
+       Returns the TSS value from the given parameters
+    """
+
+    # Define the constants (Cf is the capacity factor).
+    # Values a1 to a3 are empirically derived coefficient values.
+    capacity_factor = 0.25 if surface_type == 1 else 0.75

```

Comment:
`surface_type ==  1` doesn't really make sense anymore with `StrEnum`. It should be something like:

```suggestion
    capacity_factor = 0.25 if surface_type == SurfaceType.CONCRETE_ROOF else 0.75

```

Reply:
Perhaps
```suggestion
    roof_surface_types = {SurfaceType.CONCRETE_ROOF, SurfaceType.GALVANISED_ROOF, SurfactType.COPPER_ROOF}
    capacity_factor = 0.75 if surface_type in roof_surface_types else 0.25

```